### PR TITLE
api add babel env

### PR DESCRIPTION
**Pull request by arturparkhisenko**
In a single commit :) now, works!

### CODE REVIEW

**Comment by Kmaschta** [comment]
Good job! :+1:

File: api/.babelrc [hunk 1]
```diff
@@ -1,7 +1,14 @@
 {
-    "presets": ["es2015-node6", "es2016", "stage-2"],
+    "presets": [
+      ["env", {
+          "targets": {
+              "node": "current"
```
Comment (Kmaschta) [inline api/.babelrc]: For now, AWS Lambda can only be run with Node 6.10.

image

Can you configure babel in order to support this version, wherever we transpile the API?

File: api/.babelrc [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+      }]
+    ],
     "plugins": [
         "add-module-exports",
-        "transform-async-to-generator"
+        "transform-async-to-generator",
+        "transform-object-rest-spread"
     ]
-}
+}
```
Comment (Kmaschta) [inline api/.babelrc]: Please let the EOF char in an additional line.
(it's important for unix users)

File: api/package.json [hunk 3]
```diff
@@ -6,26 +6,23 @@
...
     "babel-plugin-transform-runtime": "6.23.0",
-    "babel-polyfill": "~6.9.1",
-    "babel-preset-es2015": "~6.6.0",
-    "babel-preset-es2015-node6": "~0.4.0",
-    "babel-preset-es2016": "6.22.0",
-    "babel-preset-stage-0": "~6.5.0",
-    "babel-register": "~6.7.2",
-    "babel-runtime": "~6.6.1",
+    "babel-polyfill": "~6.26.0",
+    "babel-preset-env": "1.6.1",
```
Comment (Kmaschta) [inline api/package.json]: In order to keep security updates, can you prefix your dependencies with `~`?

**Comment by arturparkhisenko** [comment]
@Kmaschta okay thanks for the review, I'll make changes soon.
About `"current"` it means it polyfills current running (any version, I thought it'll be better).

**Comment by Kmaschta** [comment]
In general, you're right, `current` is a better option.
But keep in mind that the code is build locally to be deployed on AWS lambda.
And in local env, you can build it with Node 8, but it's run with Node 6 on lambda.

**Pull request merged**